src/libcore/clone.rs

```diff
@@ -63,28 +63,38 @@
 /// This trait can be used with `#[derive]` if all fields are `Clone`. The `derive`d
 /// implementation of [`clone`] calls [`clone`] on each field.
 ///
+/// For a generic struct, the implementation adds necessary `Clone` bound on generic parameters.
+///
+/// ```
+/// // Reading<T> is Clone if T is Clone
```

Comment:
The type `T` does not have a `Clone` trait bound, so this comment is precise？

Reply:
Update a bit to avoid confusion.